installer/install.sh

```diff
@@ -15,39 +16,34 @@ GIT_BRANCH=${GITHUB_HEAD_REF:-`git branch --show-current`}
 echo "Set VAULT_TOKEN in a secret for vault-secrets-operator..."
 # The namespace may not exist already, but don't error if it does.
 kubectl create ns vault-secrets-operator || true
+#kubectl create secret generic vault-secrets-operator \
+#  --namespace vault-secrets-operator \
+#  --from-literal=VAULT_TOKEN=$VAULT_TOKEN \
+#  --from-literal=VAULT_TOKEN_LEASE_DURATION=31536000 \
+#  --dry-run -o yaml | kubectl apply -f -
 kubectl create secret generic vault-secrets-operator \
   --namespace vault-secrets-operator \
-  --from-literal=VAULT_TOKEN=$VAULT_TOKEN \
-  --from-literal=VAULT_TOKEN_LEASE_DURATION=31536000 \
+  --from-literal=VAULT_ROLE_ID=$(vault read --format=json auth/approle/role/rubin-data-dev.slac.stanford.edu/role-id | jq -M .data.role_id  | sed 's/"//g') \
+  --from-literal=VAULT_SECRET_ID=$(vault write -f --format=json auth/approle/role/rubin-data-dev.slac.stanford.edu/secret-id | jq .data.secret_id | sed 's/"//g') \
+  --from-literal=VAULT_TOKEN_MAX_TTL=600 \
   --dry-run=client -o yaml | kubectl apply -f -
-
-echo "Set up docker pull secret for vault-secrets-operator..."
-vault kv get --field=.dockerconfigjson $VAULT_PATH_PREFIX/pull-secret > docker-creds
-kubectl create secret generic pull-secret -n vault-secrets-operator \
-    --from-file=.dockerconfigjson=docker-creds \
-    --type=kubernetes.io/dockerconfigjson \
-    --dry-run=client -o yaml | kubectl apply -f -
-
-
+  
 echo "Update / install vault-secrets-operator..."
 # ArgoCD depends on pull-secret, which depends on vault-secrets-operator.
 helm dependency update ../services/vault-secrets-operator
 helm upgrade vault-secrets-operator ../services/vault-secrets-operator \
   --install \
-  --values ../services/vault-secrets-operator/values.yaml \
   --values ../services/vault-secrets-operator/values-$ENVIRONMENT.yaml \
   --create-namespace \
   --namespace vault-secrets-operator \
   --timeout 15m \
   --wait
 
-echo "Update / install argocd using helm..."
+echo "Update / install argocd using helm3..."
 helm dependency update ../services/argocd
 helm upgrade argocd ../services/argocd \
   --install \
-  --values ../services/argocd/values.yaml \
   --values ../services/argocd/values-$ENVIRONMENT.yaml \
-  --set global.vaultSecretsPath="$VAULT_PATH_PREFIX" \
```

Comment:
These changes are all required. I think this may have been a merge failure where you're accidentally reverting changes made by later updates to Phalanx.